plugins/inputs/syslog/README.md

```diff
@@ -55,6 +55,11 @@ Syslog messages should be formatted according to
   ## By default best effort parsing is off.
   # best_effort = false
 
+  ## The RFC standard to use for message parsing
+  ## By default RFC5424 is used. RFC3164 only supports UDP transport (no streaming support)
+  ## Must be one of "5424", or "3164".
+  # syslog_standard = "5424"
```

Comment:
Why not let the options be `RFC5424` and `RFC3164`? That makes more sense instead of just the numbers..

Reply:
Agreed, that is more clear :) I'll update shortly, thank you!